algorithms/src/Kokkos_Sort.hpp

```diff
@@ -269,9 +270,11 @@ class BinSort {
       Kokkos::parallel_for (Kokkos::RangePolicy<execution_space,bin_sort_bins_tag>(0,bin_op.max_bins()) ,*this);
   }
 
-  // Sort a view with respect ot the first dimension using the permutation array
+  // Sort a subset of a view with respect to the first dimension using the permutation array
   template<class ValuesViewType>
-  void sort( ValuesViewType const & values) const
+  void sort( ValuesViewType const & values
+           , int values_range_begin
```

Comment:
why are these `int` but their difference is `size_t` below?

Reply:
Excellent question, passing the buck to @crtrott . The existing code was like that (`int` arguments and `size_t` span), I just followed the convention.

Reply:
ugh, both the sizes _and_ the signedness aren't right :(

Reply:
Agreed, its fishy